### ISSUE TITLE
load_plugin_textdomain is missing.

### ISSUE THREAD

**Issue by mayukojpn** [WP-API/WP-API#2806]
We need `load_plugin_textdomain` to recognize text domain for translation editor like Poedit or GlotPress.

**Comment by websupporter** [comment]
Hi @mayukojpn,
thanks so much for your effort and contributing. It is really appreciated. 

But I think, since this plugin is supposed to merge into core we do not need a textdomain.

**Comment by mayukojpn** [comment]
Hi @websupporter 
Jorbin asked us to test internationalization on Slack polyglots channel, but we need text domain to do that.
https://wordpress.slack.com/archives/polyglots/p1476112154002294

**Comment by websupporter** [comment]
Ah sorry. I didn't get that. If you need any help, hit me on Slack :)

**Comment by websupporter** [comment]
But right now, we do not use the textdomain `rest-api`. We just use `__( 'Translate' )`... Is this a problem?

See for example here: https://github.com/WP-API/WP-API/blob/develop/lib/endpoints/class-wp-rest-attachments-controller.php#L45

**Comment by mayukojpn** [comment]
Yes, We also need add textdomain after strings. Otherwise WordPress will use core translation file. I'm also working for that!

**Comment by miya0001** [comment]
I feel that textdomain should not be added to this plugin, because we have to remove them after merging into WordPress core.
We need to try to generate `.po` with WordPress core, so we can translate them. 
Maybe it is not difficult.

@mayukojpn May I do it?

**Comment by mayukojpn** [comment]
Thank you @miya0001 ! Can you talk with Jorbin about that?

**Comment by miya0001** [comment]
Hi,
I developed a plugin that allowed you to translate temporarily rest-api plugin.
If you activate this plugin, you don't need to change textdomain or so.

Thanks!

https://github.com/miya0001/wp-api-i18n

**Comment by kadamwhite** [comment]
Milestoning this as 2.1 because text domains can be added once the go/no-go on 4.7 is received. Thanks for your assistance @miya0001, this is hugely helpful!
